src/index.ts

```diff
@@ -1,6 +1,9 @@
 import { Duplex } from 'stream';
 import { Runtime } from 'webextension-polyfill-ts';
 
+const browser: any  = globalThis.browser || globalThis.chrome;
+const runtime: object = browser.runtime || {};
```

Comment:
The "static" values in the runtime API are exposed under the "Static" property:
```suggestion
const runtime: Runtime.Static = browser.runtime || {};
```